source/code/plugins/scom_common.rb

```diff
@@ -66,7 +66,13 @@ def self.parse_json_record_encoding(record)
     def self.create_secure_http(uri)
       http = Net::HTTP.new( uri.host, uri.port )
       http.use_ssl = true
-      http.verify_mode = OpenSSL::SSL::VERIFY_NONE
+      if SCOM::Configuration.enable_server_auth
+        OMS::Log.info_once("Enabling server certificate validation")
+        http.verify_mode = OpenSSL::SSL::VERIFY_PEER
+      else
+        OMS::Log.info_once("Disabling server certificate validation")
```

Comment:
Out of curiosity, have you watched what happens if the using switches their enable_server_auth parameter more than once?

For example, if they have the default, and run for a while;
then they set it to true and restart OMSAgent to take effect;
then they set it back to false and restart OMSAgent to take effect....

Does it log three times, which includes logging "Disabling server certificate validation" twice?